components/forms/Form.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Formik, Form as FormikForm } from 'formik';
+import PropTypes from 'prop-types';
+
+const Form = ({ children, className, form, path }) => (
+    <Formik
+        initialValues={{}}
+        onSubmit={async values => {
+            const res = await fetch(
+                // FIXME: use process.env.API_ENDPOINT
+                `http://localhost:5000/forms/${path}`,
+                {
+                    body: JSON.stringify(values),
+                    headers: {
+                        'Content-Type': 'application/json'
+                    },
+                    method: 'POST'
+                }
+            )
+            try {
+                const result = await res.json()
+                console.log(result)
```

Comment:
Definitely not blocking (and we could do a pass for all log statements at some point later on), but we might want to remove this `console.log` statement.

```suggestion
```

Reply:
Done!